extensions/oidc/deployment/src/main/java/io/quarkus/oidc/deployment/OidcBuildStep.java

```diff
@@ -320,6 +320,13 @@ private static boolean isTenantIdentityProviderType(InjectionPointInfo ip) {
         return TENANT_IDENTITY_PROVIDER_NAME.equals(ip.getRequiredType().name());
     }
 
+    @Record(ExecutionTime.RUNTIME_INIT)
+    @Consume(RuntimeConfigSetupCompleteBuildItem.class)
+    @BuildStep
```

Comment:
I'd expect there will be a race between the `TenantConfigBean` and this handler. We will see if CI is ok, but I wonder what would happen if added here `@Consume(SyntheticBeansRuntimeInitBuildItem.class)` ? if there is a race, which it might, we will need to think about another solution. please try it and report back.